xml/System.IO.Compression/DeflateStream.xml

```diff
@@ -58,10 +58,24 @@
     <remarks>
       <format type="text/markdown"><![CDATA[
 
-[!INCLUDE[remarks](~/includes/remarks/System.IO.Compression/DeflateStream/DeflateStream.md)]
+This class represents the Deflate algorithm, which is an industry-standard algorithm for lossless file compression and decompression. Starting with .NET Framework 4.5, the <xref:System.IO.Compression.DeflateStream> class uses the zlib library. As a result, it provides a better compression algorithm and, in most cases, a smaller compressed file than it provides in earlier versions of .NET Framework.
```

Comment:
Where includes were only used once (i.e. weren't shared), I brought the content back inline since separate include files in the dotnet-apid-docs is not the direction that the .NET libraries team wants to take going forward. They want all docs source to live in dotnet/runtime, and having it inline will make that porting easier.